package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-workspace",
-  "version": "3.42.0",
+  "version": "4.0.0",
```

Comment:
We usually use a pre-release version number in a patch like this (e.g. `4.0.0-alpha.1`) and then let `slt-release` change it to the "production" version number.

Reply:
@bajtos done PTAL